README.md

```diff
@@ -47,3 +47,50 @@ We are replacing BioMart with a more scalable and performant solution that allow
 ### Global Search
 
 Similar to our current site-wide search, the Global Search will allow you to search for gene names and symbols and other identifiers.
+
```

Comment:
Delete line 21 above, which references the old Readme.

At some later point, we'll review the Readme.